manager/eris-mint/evm/snative_test.go

```diff
@@ -0,0 +1,124 @@
+package vm
+
+import (
+	"encoding/hex"
+	"testing"
+
+	. "github.com/eris-ltd/eris-db/word256"
+	"github.com/stretchr/testify/assert"
+	. "github.com/eris-ltd/eris-db/manager/eris-mint/evm/opcodes"
+	ptypes "github.com/eris-ltd/eris-db/permission/types"
+)
+
+/* Compiling the Permissions solidity contract at
+(generated by Solidity() function)
+https://ethereum.github.io/browser-solidity yields:
+
+Functions
+3fbf7da5 add_role(address,bytes32)
```

Comment:
Need to decide if we're going CamelCase Here...or mixedCase (as is Solidity convention).